maturity.go

```diff
@@ -66,6 +66,7 @@ func (client *Client) ListServicesMaturity(variables *PayloadVariables) (*Servic
 		}
 		q.Account.Services.Nodes = append(q.Account.Services.Nodes, resp.Nodes...)
 		q.Account.Services.PageInfo = resp.PageInfo
+		q.Account.Services.TotalCount += resp.TotalCount
 	}
```

Comment:
all you actually need to do is this
```suggestion
	}
	q.Account.Services.TotalCount = len(q.Account.Services.Nodes)
```